linalg/sundials.cpp

```diff
@@ -2266,18 +2370,21 @@ void KINSolver::Mult(Vector &x,
 
    if (rank == 0)
    {
+#if MFEM_SUNDIALS_VERSION < 70000
       flag = KINSetPrintLevel(sundials_mem, print_level);
       MFEM_VERIFY(flag == KIN_SUCCESS, "KINSetPrintLevel() failed!");
+#endif
+      // NOTE: there is no KINSetPrintLevel in SUNDIALS v7!
```

Comment:
For any users that were setting the print level before, this is now a compile time option and they should set the desired logging level when configuring SUNDIALS.